ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.jsonschema.json

```diff
@@ -11335,6 +11335,14 @@
               },
               "title": "DSCP"
             },
+            "exp": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "description": "Example \"0 to traffic-class 0\"\n"
+              },
+              "title": "Exp"
```

Comment:
Please add exp -> EXP in `ansible-avd/ansible_collections/arista/avd/plugins/plugin_utils/schema/key_to_display_name.py`.